app/views/admin/contact_translations/edit.html.erb

```diff
@@ -0,0 +1,205 @@
+<% content_for :context, @contact.title %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @contact, parent_class: "contact")) %>
+<% content_for :page_title, "Edit translation for: #{@contact.title}" %>
+<% content_for :title, "Edit #{@translation_locale.native_language_name} (#{@translation_locale.english_language_name}) translation" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @translated_contact, url: admin_organisation_contact_translation_path(@contactable, @translated_contact, translation_locale), method: :put do |form| %>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Title (required)",
+        },
+        heading_size: "l",
+        name: "contact[title]",
+        id: "contact_title",
+        value: @translated_contact.title,
+        error_items: errors_for(form.object.errors, :title),
+        right_to_left: @translated_contact.translation_locale.rtl?,
+        right_to_left_help: false
+      } %>
+
+      <h2 class="app-view-translation__english-content govuk-heading-m govuk-!-margin-bottom-2">English
+        translation:</h2>
+      <p class="app-view-translation__english-content govuk-body"><%= @translated_contact.title %></p>
+
+      <%= render "govuk_publishing_components/components/textarea", {
+        label: {
+          heading_size: "l",
+          text: "Comments",
+        },
+        name: "contact[comments]",
+        id: "contact_comments",
+        value: @translated_contact.comments,
+        error_items: errors_for(form.object.errors, :comments),
+        right_to_left: @translated_contact.translation_locale.rtl?,
+        right_to_left_help: false
+      } %>
+
+      <h2 class="app-view-translation__english-content govuk-heading-m govuk-!-margin-bottom-2">English
+        translation:</h2>
+      <p class="app-view-translation__english-content govuk-body"><%= @translated_contact.comments %></p>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Recipient (required)",
```

Comment:
I think the only required fields on this are title and street address 
<img width="797" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/f2e74dc5-6bba-48ce-ac5d-c4d4e699edc2">